src/main/java/Duke/Parser.java

```diff
@@ -0,0 +1,89 @@
+package duke;
+
+import duke.exception.*;
```

Comment:
I think wildcard imports are not allowed according to the module coding standards?